nixos/modules/services/networking/zerobin.nix

```diff
@@ -88,7 +88,7 @@ in
         enable = true;
         after = [ "network.target" ];
         wantedBy = [ "multi-user.target" ];
-        serviceConfig.ExecStart = "${pkgs.pythonPackages.zerobin}/bin/zerobin ${cfg.listenAddress} ${toString cfg.listenPort} false ${cfg.user} ${cfg.group} ${zerobin_config}";
+        serviceConfig.ExecStart = "${pkgs.zerobin}/bin/zerobin ${cfg.listenAddress} ${toString cfg.listenPort} false ${cfg.user} ${cfg.group} ${zerobin_config}";
```

Comment:
Oh! If zerobin is considered a `networking/` service, then the package definition can be also under `pkgs/applications/networking/`.

Reply:
@ento I've moved to `zerobin` to `pkgs/applications/networking/` as you suggested